doc/source/ecosystem.rst

```diff
@@ -153,6 +153,11 @@ A good implementation for Python users is `has2k1/plotnine <https://github.com/h
 Spun off from the main pandas library, the `qtpandas <https://github.com/draperjames/qtpandas>`__
 library enables DataFrame visualization and manipulation in PyQt4 and PySide applications.
 
+`D-Tale <https://github.com/man-group/dtale>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+D-Tale is the combination of a Flask back-end and React front-end to bring you an easy way to view, edit & analyze Pandas data structures. Also, includes a fully-featured chart builder by way of Plotly Dash. It integrates seamlessly with jupyter notebooks, python terminals, kaggle & Google Colab. Currently this tool supports such Pandas objects as DataFrame, Series, MultiIndex, DatetimeIndex & RangeIndex. A nice demo is located `here <http://alphatechadmin.pythonanywhere.com/>`__.
```

Comment:
About the format:
- Can you write this in multiple lines, like 80 characters long or so.
- Can you make `pandas` lowercase everywhere, is how we capitalize it.

I don't know about D-Tale, but I don't find this description very useful. I think it goes into details like that it uses flask and react, that it works in notebooks, or the data types, but after reading this I still have no idea why it can be useful to me.

Clicking in the demo feels like a spreadsheet UI for pandas dataframes. Do you mind rephrasing this so users understand how this can be useful for them, instead of writing implementation details.

Reply:
@datapythonista I reworked the description and hopefully its a little clearer now but maybe too verbose.  One question I had was on the little code snippet to invoke D-Tale.  Should I keep it or drop it?  I know its ugly with the import and execution in one-line but I didn't want to take up more space.  Thanks